app/index.html

```diff
@@ -56,5 +56,7 @@
     <script src="script/controllers/main.controller.js" type="text/javascript"></script>
     <script src="script/controllers/categories.controller.js" type="text/javascript"></script>
     <script src="script/controllers/categories.details.controller.js" type="text/javascript"></script>
+    <script src="script/directives/sortable.list.directive.js" type="text/javascript"></script>
+    <script src="script/directives/template_and_js_for_sortable_list_directive/sortable.list.controller.js" type="text/javascript"></script>
```

Comment:
I think the folder name should be sortable.list, and you can put all the files which belongs to this directive.
(btw we usually don't create another file to a controller, it can be in the directives file)